python/delta_sharing/delta_sharing.py

```diff
@@ -99,6 +107,8 @@ def load_as_spark(url: str, version: Optional[int] = None) -> "PySparkDataFrame"
     df = spark.read.format("deltaSharing")
     if version is not None:
         df.option("versionAsOf", version)
+    if timestamp is not None:
+        df.option("timestampAsOf", timestamp)
```

Comment:
Question: If the caller specifies both, are we relying on the server to return error?

Reply:
there's a test case in test_delta_sharing.py L623, it's checked in the deltaSharingDataSource, still at the client side.